lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -634,6 +634,21 @@ describe('ContentNodeSelectorComponent', () => {
                 expect(component.showingSearchResults).toBeFalsy();
             });
 
+            it('should restrict the search to the root in case is defined', () => {
+                component.root = 'my-root-id';
+                component.ngOnInit();
+                component.search('search');
+                expect(cnSearchSpy).toHaveBeenCalledWith('search', 'my-root-id', 0, 25, [], false);
```

Comment:
can you separate the rest from the expectation with an empty line? Style 05-16 https://w3.alfresco.com/confluence/pages/viewpage.action?spaceKey=AP&title=Code+Style+E2E